fiberzap/zap.go

```diff
@@ -81,8 +82,25 @@ func New(config ...Config) fiber.Handler {
 			stop = time.Now()
 		}
 
+		// Check if the logger has the appropriate level
+		s := c.Response().StatusCode()
+		var ce *zapcore.CheckedEntry
+		switch {
+		case s >= 500:
+			ce = cfg.Logger.Check(cfg.Levels[0], cfg.Messages[0])
+		case s >= 400:
+			ce = cfg.Logger.Check(cfg.Levels[1], cfg.Messages[1])
+		default:
+			ce = cfg.Logger.Check(cfg.Levels[2], cfg.Messages[2])
+		}
+
+		if ce == nil {
```

Comment:
Do we really need this?

Reply:
the Check func will return a nil logger if the level is lower, than the configured logger level, so yes, we need a check here.

Reply:
Oh right